src/Core/Extensions/MessageExtensions.cs

```diff
@@ -2,6 +2,7 @@
 using System.Threading.Tasks;
 using System.Collections.Generic;
 using System.Text;
+using System.Text.RegularExpressions;
```

Comment:
Is this the only change in the file? If so, might want to undo it.

Reply:
I'll undo it. I initially had a method in the file that used it, but then I moved it and forgot the remove the import statement.